lib/yubihsm_winhttp.c

```diff
@@ -296,9 +296,7 @@ static yh_rc backend_send_msg(yh_backend *connection, Msg *msg, Msg *response,
   DWORD dwStatusCode = 0;
   DWORD dwSize = sizeof(dwStatusCode);
   struct context *context = calloc(1, sizeof(struct context));
-  wchar_t hsm_identifier[129];
-  wchar_t *tmp_identifier;
-  size_t tmp_identifier_len;
+  wchar_t hsm_identifier[64];
```

Comment:
Could we initialize the hsm_identifier array to 0s so that if, for whatever reason, swprintf fails the conversion, we will not be printing uninitialized stack memory?

That or/and check the return value from swprintf

Reply:
We have many instances where we assume sprintf etc will write something, they should all be fixed in that case

Reply:
yeah, just thought we were improving the code :P

Reply:
Yes, but that wasn't really checked for in the old code either.